### DISCUSSION TITLE
how to remove a dep that was added by hand to a dev env

### DISCUSSION THREAD

**Discussion by j1philli** [pkgxdev/pkgx#801] [Help] [answered]
I added the wrong yarn to a dev environment but can't easily figure out how to remove it

**Comment by jhheider** [comment, marked as answer]
```shell
$ env +yarnpkg.com
 ✓  ~/.pkgx/yarnpkg.com/v3.6.4
$ which yarn
/Users/jacob/.pkgx/yarnpkg.com/v3.6.4/bin/yarn
$ env -yarnpkg.com
which yarn
yarn not found
```

**Reply by j1philli** [reply]
ah it does work temporarily
it gets added back when dev is refreshed

**Reply by j1philli** [reply]
and now it doesn't when I try again. very odd

**Reply by j1philli** [reply]
ah I figured it out. it doesn't work if the package is a dependency of another install pkgx package. made this for it https://github.com/pkgxdev/pkgx/issues/806
